src/components/furniture/Separators.vue

```diff
@@ -27,21 +27,21 @@
         :style="{
           width: separatorWidth + '%',
         }"
-        v-if="amountOfSeparators === '3'"
+        v-if="separators.amountOfSeparators === '3'"
       ></div>
       <div
         class="separator"
         :style="{
           width: separatorWidth + '%',
         }"
-        v-if="amountOfSeparators === '3'"
+        v-if="separators.amountOfSeparators === '3'"
       ></div>
       <div
         class="separator"
         :style="{
           width: separatorWidth + '%',
         }"
-        v-if="amountOfSeparators === '3'"
+        v-if="separators.amountOfSeparators === '3'"
```

Comment:
it's weird that it should be equal to a string `javascript '3' ` instead of the number 3. If it's because it's a human input, you should convert it at the moment tu user submit it, but not keep a number disguised as a string as it could lead to problems (imagine one day you forget that, you want to do `separators.amountOfSeparators = separators.amountOfSeparators + 1;` en you end up with a string '31' instead of 4